SourceX/DiabloUI/selgame.cpp

```diff
@@ -218,6 +218,7 @@ void selgame_Diff_Esc()
 void selgame_Password_Init(int value)
 {
 	memset(&selgame_Password, 0, sizeof(selgame_Password));
+	getIniValue("Phone Book", "Password1", selgame_Password, 15);

```

Comment:
For now this is a bit of a workaround for not having a onscreen keyboard, so I will consider this a draft PR (not ready to merge, but keeping track of progress)

Reply:
Yes, that's fine as a draft first.